src/interface/TWSolanaToken.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include <TrustWalletCore/TWSolanaToken.h>
+#include "Solana/Address.h"
+
+using namespace TW::Solana;
+using namespace TW;
+
+/// Wrapper for C interface, empty as all methods are static
+struct TWSolanaToken {
+};
+
+TWString *_Nonnull TWSolanaTokenDefaultTokenAddress(TWString *_Nonnull mainAddress, TWString* _Nonnull tokenMintAddress) {
```

Comment:
let's change it to an instance method instead? so app can use `SolanaAddress` instead of `AnyAddress`

Reply:
I don't understand, can you explain (maybe in private)

Reply:
I thought it might be nicer to have a `TWSolanaAddress` interface, a bit like AnyAddress but app side can use method like:

```swift
let address = SolanaAddress(string: <main address>)
let tokenAddress = address.defaultToken(address: <tokken mint address>
```

Reply:
Changed.